### PR TITLE
[13.0][MIG]pos_product_sort: standard migration

### PR DESCRIPTION
**Pull request by manuelregidor**

### CODE REVIEW

File: pos_product_sort/__manifest__.py [hunk 1]
```diff
@@ -3,16 +3,20 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 {
-    "name": "Point of Sale - Products Sorted by Name",
-    "version": "12.0.1.0.1",
-    "category": "Point Of Sale",
-    "summary": """
+    'name': 'Point of Sale - Products Sorted by Name',
```
Comment (chienandalu) [inline pos_product_sort/__manifest__.py]: Why are you reverting pre-commit changes? :thinking:

**Review by chienandalu — CHANGES_REQUESTED** [review]

**Comment by legalsylvain** [comment]
hi @manuelregidor thanks for porting this module. could you take a look on travis that is red ? 

``pre-commit run -a`` should fix the problem I guess.

regards.

**Review by chienandalu — APPROVED** [review]
Thanks :+1:

**Review by legalsylvain — APPROVED** [review]
thanks :+1: 
diff review + test on runbot. LGTM.

**Review by ValentinVinagre — APPROVED** [review]

**Review by HaraldPanten — APPROVED** [review]
LGTM 👍 

@legalsylvain @chienandalu is it OK to merge for fast tracking?

**Pull request merged**